Create Executive vote - April 18, 2024.md (#2620)
makerdao/community: governance/votes/Executive vote - April 18, 2024.md

---
title: Template - [Executive Vote] Aligned Delegate Compensation, AVC Member Compensation, Aave Revenue Share Payment, Whitelist New Address in the RWA015-A Output Conduit, Push USDP out of Input Conduit, Trigger Spark Proxy Spell - April 18, 2024

> summary: Aligned Delegate and AVC member compensation, Spark Protocol-Aave Share Payment, Whitelisting new address in the RWA015-a output conduit, pushing 84,211.27 USDP out of the MCD PSM PAX input conduit jar, triggering a Spark proxy spell on Ethereum and Gnosis.

Reviewer: ```suggestion
summary:  March 2024 AD Compensation, Q1 2024 AVC Member Participation Reward, Spark Protocol-Aave Share Transfer, Whitelisting new address in the RWA015-a output conduit, pushing 84,211.27 USDP out of the MCD PSM PAX input conduit jar, triggering a Spark proxy spell on Ethereum and Gnosis.
```

date: 2024-04-18T00:00:00.000Z
address: "$spell_address"

---

> # [Executive Proposal] [Executive Vote] Aligned Delegate Compensation, AVC Member Compensation, Aave Revenue Share Payment, Whitelist New Address in the RWA015-A Output Conduit, Push USDP out of Input Conduit, Trigger Spark Proxy Spell - April 18, 2024

Reviewer: ```suggestion
# [Executive Proposal] [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Reward, Aave Revenue Share Transfer, Whitelist New Address in the RWA015-A Output Conduit, Push USDP out of Input Conduit, Trigger Spark Proxy Spell - April 18, 2024
```

> The Governance Facilitators and SideStream have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

Reviewer: ```suggestion
The Governance Facilitators, SideStream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
```

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:

- A total of **219.36 MKR** will be distributed to eight Aligned Delegates as compensation for March 2024.
- A total of **175.14 MKR** will be distributed to nine AVC members as compensation for March 2024.
- A total of **238,339 DAI** will be paid to Aave as part of the SparkLend revenue share.
- A new address in the RWA015-A output conduit will be whitelisted.

> - **84,211.27 USDP** will be pushed out of the  MCD PSM PAX Input Conduit Jar.

Reviewer: ```suggestion
- **84,211.27 USDP** will be pushed out of the MCD PSM PAX Input Conduit Jar.
```

- A proxy spell for Spark Protocol on Ethereum and Gnosis will be triggered.

**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

> Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.

Reviewer: ```suggestion
Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
```

- **Proposal:** [Forum post](https://forum.makerdao.com/t/march-2024-aligned-delegate-compensation/24088)

If this executive proposal passes, the following transfers totalling **219.36 MKR** for Aligned Delegate Compensation will be made to qualifying Aligned Delegates.

| Delegate | Address | Amount (MKR) |
|--|--|--|
| BONAPUBLICA | 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3 | 41.67 |
| Cloaky | 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818 | 41.67 |
| TRUE NAME | 0x612F7924c367575a0Edf21333D96b15F1B345A5d | 41.67 |
| BLUE | 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf | 39.75 |
| vigilant | 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61 | 13.89 |
| Pipkin | 0x0E661eFE390aE39f90a58b04CF891044e56DEDB7 | 13.89 |
| JAG | 0x58D1ec57E4294E4fe650D1CB12b96AE34349556f | 13.89 |
| UPMaker | 0xbB819DF169670DC71A16F58F55956FE642cc6BcD | 12.93 |

### AVC Members Compensation Q1 2024

- **Authorization:** [Atlas Section 2.5.10](https://mips.makerdao.com/mips/details/MIP101#2-5-10-avc-member-participation-rewards)
- **Proposal:** [Forum post](https://forum.makerdao.com/t/avc-member-participation-rewards-q1-2024/24083)

> If this executive proposal passes, the following distributions totalling **175.14 MKR** will be made for Q4 AVC member compensation.

Reviewer: ```suggestion
If this executive proposal passes, the following distributions totalling **175.14 MKR** will be made for Q1 2024 AVC member compensation.
```

---

> title: Template - [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Reward, Aave Revenue Share Transfer, Whitelist New Address in the RWA015-A Output Conduit, Push USDP out of Input Conduit, Trigger Spark Proxy Spell - April 18, 2024

Reviewer 2: :green_circle: **nit. non-blocking**

1. Aligns title with previous conventions
2. Minor grammatical correction

```suggestion
title: Template - [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Rewards, Aave-SparkLend Revenue Share, Whitelist New Address in the RWA015-A Output Conduit, USDP Input Conduit Managment, Spark Proxy Spell - April 18, 2024
```

> summary:  March 2024 AD Compensation, Q1 2024 AVC Member Participation Reward, Spark Protocol-Aave Share Transfer, Whitelisting new address in the RWA015-a output conduit, pushing 84,211.27 USDP out of the MCD PSM PAX input conduit jar, triggering a Spark proxy spell on Ethereum and Gnosis.

Reviewer 2: :green_circle: **nit. non-blocking**

This isn't really a summary, rather it is a repeat of the title. It could also benefit from using active voice. Suggested improvement below.

```suggestion
summary:  Distribute AD compensation for March 2024 and AVC Member participation rewards for Q1 2024, transfer Dai to Aave as part of the SparkLend revenue sharing agreement, whitelist a new address in the RWA015-a output conduit, convert USDP to Dai and deposit to the Surplus Buffer, trigger a Spark proxy spell on Ethereum and Gnosis.
```

date: 2024-04-18T00:00:00.000Z
address: "$spell_address"

---

> # [Executive Proposal] [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Reward, Aave Revenue Share Transfer, Whitelist New Address in the RWA015-A Output Conduit, Push USDP out of Input Conduit, Trigger Spark Proxy Spell - April 18, 2024

Reviewer 2: :yellow_circle: **moderate**

1. Remove unneeded text
2. Align title with suggestion from line 1

```suggestion
# [Executive Proposal] March 2024 AD Compensation, Q1 2024 AVC Member Participation Rewards, Aave-SparkLend Revenue Share, Whitelist New Address in the RWA015-A Output Conduit, USDP Input Conduit Managment, Spark Proxy Spell - April 18, 2024
```

The Governance Facilitators, SideStream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:

> - A total of **219.36 MKR** will be distributed to eight Aligned Delegates as compensation for March 2024.

Reviewer 2: :red_circle: **blocking**

This value is incorrect and does not match the total in line 42.

```suggestion
- A total of **214.55 MKR** will be distributed to eight Aligned Delegates as compensation for March 2024.
```

- **84,211.27 USDP** will be pushed out of the MCD PSM PAX Input Conduit Jar.
- A proxy spell for Spark Protocol on Ethereum and Gnosis will be triggered.

**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**

Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.

---

## Proposal Details

### Aligned Delegate Compensation for March 2024

- **Authorization:** [Atlas Section 2.6.3](https://mips.makerdao.com/mips/details/MIP101#2-6-3-aligned-delegate-income-and-participation-requirements)
- **Proposal:** [Forum post](https://forum.makerdao.com/t/march-2024-aligned-delegate-compensation/24088)

> If this executive proposal passes, the following transfers totalling **219.36 MKR** for Aligned Delegate Compensation will be made to qualifying Aligned Delegates.

Reviewer: ```suggestion
If this executive proposal passes, the following transfers totalling **214.55 MKR** for Aligned Delegate Compensation will be made to qualifying Aligned Delegates.
```

| Delegate | Address | Amount (MKR) |
|--|--|--|
| BONAPUBLICA | 0x167c1a762B08D7e78dbF8f24e5C3f1Ab415021D3 | 41.67 |
| Cloaky | 0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818 | 41.67 |
| TRUE NAME | 0x612F7924c367575a0Edf21333D96b15F1B345A5d | 41.67 |
| BLUE | 0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf | 39.75 |
| vigilant | 0x2474937cB55500601BCCE9f4cb0A0A72Dc226F61 | 13.89 |
| Pipkin | 0x0E661eFE390aE39f90a58b04CF891044e56DEDB7 | 13.89 |

> | JAG | 0x58D1ec57E4294E4fe650D1CB12b96AE34349556f | 13.89 |

Reviewer: ```suggestion
| JAG | 0x58D1ec57E4294E4fe650D1CB12b96AE34349556f | 9.08 |
```

> | UPMaker | 0xbB819DF169670DC71A16F58F55956FE642cc6BcD | 12.93 |

Reviewer 2: :green_circle: **general comment, no action required**

In general we feel that these distributions should be ordered by amount to be distributed (largest-to-smallest) and then by Delegate Name (A-Z).

However, this order matches the sheet which is what the code will be based on so we will not suggest a change at this time in order that it matches the code for ease of review.

> | UPMaker | 0xbB819DF169670DC71A16F58F55956FE642cc6BcD | 12.93 |

Reviewer 2: :yellow_circle: **moderate**

Add Etherscan links for addresses as in previous spells.

### AVC Members Compensation Q1 2024

- **Authorization:** [Atlas Section 2.5.10](https://mips.makerdao.com/mips/details/MIP101#2-5-10-avc-member-participation-rewards)
- **Proposal:** [Forum post](https://forum.makerdao.com/t/avc-member-participation-rewards-q1-2024/24083)

If this executive proposal passes, the following distributions totalling **175.14 MKR** will be made for Q1 2024 AVC member compensation.

| AVC Member | Address | Amount (MKR) |
|--|--|--|
| IamMeeoh | 0x47f7A5d8D27f259582097E1eE59a07a816982AE9 | 20.85  |
| DAI-Vinci | 0x9ee47F0f82F1A6F45C4E1D25Ce95C321D8C8356a | 20.85  |
| opensky | 0xf44f97f4113759E0a57756bE49C0655d490Cf19F | 20.85  |
| ACRE DAOs | 0xBF9226345F601150F64Ea4fEaAE7E40530763cbd | 20.85  |
| Res | 0x8c5c8d76372954922400e4654AF7694e158AB784 | 20.85  |
| Harmony | 0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3 | 20.85  |
| Libertas | 0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f | 20.85  |
| seedlatam.eth | 0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299 | 20.85  |

> | 0xRoot | 0xC74392777443a11Dc26Ce8A3D934370514F38A91 | 8.34  |

Reviewer 2: :yellow_circle: **moderate**

Add Etherscan links for addresses as in previous spells.

### Aave Revenue Share Payment

- **Authorization:** [Support Scope Section 9.4.1](https://mips.makerdao.com/mips/details/MIP106#9-4-1-spark-protocol-aave-revenue-share)
- **Proposal:** [Forum post](https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-3-q1-2024/24014)

If this executive proposal passes, a payment from the Surplus Buffer of **238,339 DAI** to a smart contract ([0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c)) controlled by Aave Governance will be made as part of the Spark Protocol-Aave Revenue Share.

> ### Whitelisting New Address in the RWA015-A Output Conduit

Reviewer 2: :green_circle: **nit**

```suggestion
### Whitelist New Address in the RWA015-A Output Conduit
```

- **Authorization:** [Atlas Section 2.5.10](https://mips.makerdao.com/mips/details/MIP101#2-5-10-avc-member-participation-rewards)

> - **Proposal:** [Forum post](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-04-18/24084)

Reviewer 2: :red_circle: **blocking**

1. Detail of address to whitelist is missing
2. Authorization link is incorrect

If this executive proposal passes, following the recommendation of the Governance Facilitators in collaboration with the spell team, the following operations will be performed:

- Raise the PSM_PAX_A Debt Ceiling to **100,000 DAI**.
- Call `push(uint256 amt)` on the the PSM_PAX_A_INPUT_CONDUIT_JAR contract with an amount of **84,211.27 USDP**, depositing the contents of the contract to the PSM and receiving an identical amount of Dai into the PSM_PAX_A_JAR contract.
- Call `void()` on the PSM_PAX_A_JAR contract to deposit the Dai to the Surplus Buffer.
- Set PSM-PAX-A Debt Ceiling to **0 DAI**.
- Decrease Debt Ceiling of PSM_PAX_A back to **0 DAI**.

This will have the following net results:

- The PSM_PAX_A Debt Debt Ceiling remains at 0 DAI after the transaction has been executed.
- Dai generated from PSM_PAX_A increases by **84,211.27 DAI**.
- USDP balance of MCD_JOIN_PSM_PAX_A increases by **84,211.27 USDP**.
- USDP balance of PSM_PAX_A_INPUT_CONDUIT_JAR decreases to **0 USDP**.

> - Surplus Buffer increases by **84,211.27 DAI**.

Reviewer 2: :red_circle: **blocking**

Section does not have a correct header

> ### Triggering Spark Proxy Spell

Reviewer 2: :green_circle: **nit**

```suggestion
### Trigger Spark Proxy Spell
```

- **Proposal:** [Forum post "[Apr 4, 2024] Proposed Changes to SparkLend for Upcoming Spell"](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033)

If this executive proposal passes, the SparkLend Proxy Spell on Ethereum and Gnosis at [0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882](https://etherscan.io/address/0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882) will be executed, containing the following items:

- [Ethereum+Gnosis] Upgrade Pool Implementation to Disable Flashloan into Borrow Feature
- [Gnosis] Onboard sxDAI
- [Gnosis] Onboard EURe
- [Gnosis] Onboard USDC
- [Gnosis] Onboard USDT
- [Gnosis] Refresh Existing Market Parameters
- [Gnosis] Update xDAI Oracle Parameter

> - [Ethereum] Activate the SparkLend External Security Access Multisig

Reviewer 2: :red_circle: / :yellow_circle: **moderate/severe**

We support itemizing inclusions for Spark Proxy spells as voters still need to understand the details of their votes.

See: https://github.com/makerdao/community/blob/master/governance/votes/Executive%20Vote%20-%20April%204%2C%202024.md#trigger-sparklend-proxy-spell

- **Proposal:** [Forum post "[Apr 4, 2024] Proposed Changes to SparkLend for Upcoming Spell"](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033)

If this executive proposal passes, the SparkLend Proxy Spell on Ethereum and Gnosis at [0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882](https://etherscan.io/address/0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882) will be executed, containing the following items:

#### [Ethereum+Gnosis] Upgrade Pool Implementation to Disable Flashloan into Borrow Feature

This item upgrades the SparkLend pool implementation to disable the flashloan-to-borrow feature, reducing unnecessary risks and aligning with the latest Aave V3 release standards for enhanced security.

####  [Gnosis] Onboard sxDAI

> This item proposes to onboard sxDAI, a yield-enhanced version of sDAI on the Gnosis Chain, as collateral.

Reviewer: ```suggestion
This item proposes to onboard sxDAI (Savings xDAI), a yield-enhanced version of sDAI on the Gnosis Chain, as collateral.
```

#### [Gnosis] Onboard EURe

> This item proposes to onboard EURe as collateral on the Gnosis Chain.

Reviewer: ```suggestion
This item proposes to onboard EURe (Monerium EUR emoney) as collateral on the Gnosis Chain.
```

#### [Gnosis] Onboard USDC

This item proposes to onboard USDC as collateral on the Gnosis Chain.

#### [Gnosis] Onboard USDT

> This item proposes to onboard USDT as collateral on the Gnosis Chain.

Reviewer: ```suggestion
This item proposes to onboard USDT (Tether USDT) as collateral on the Gnosis Chain.
```

#### [Gnosis] Refresh Existing Market Parameter

This item updates Gnosis market parameters, including a significant adjustment of the xDAI interest rate model to align with current conditions and the introduction of sxDAI as collateral.

> #### [Gnosis] Update xDAI Oracle Parameter

Reviewer 2: We would include parameters. This is a simple copy-paste from the poll text.

Reviewer: Considering this is the SubDao proxy spell, I'm not entirely sure if this will be beneficial for the voters or if it might make the Executive Summary too lengthy and difficult to read. The onboarding parameters have been approved and are available for review in the forum thread. I'll include them this time and see how they are displayed in the voting portal

Reviewer 2: We actually feel very strongly about this. This cannot execute without MKR voters supporting it. They are taking responsibility and potentially liability when they vote to execute the proposal. The least we can do is make sure they have the parameters they are voting to enact readily available so that they can verify the underlying code if they so wish.

This item suggests changing the oracle for xDAI from the Chainlink DAI price feed to a fixed 1:1 USD pricing.

> #### [Ethereum] Activate the SparkLend External Security Access Multisig

Reviewer 2: add link to forum post - https://forum.makerdao.com/t/sparklend-external-security-access-multisig-for-freezer-mom/24070

> This item activates the SparkLend External Security Access Multisig, which enables emergency interventions in extreme cases like potential code exploits threatening the protocol. Governed by selected members from the community, this feature aims to provide quick responses to emergencies, complementing existing governance processes.

Reviewer 2: add multisig address?

| ACRE DAOs | [0xBF9226345F601150F64Ea4fEaAE7E40530763cbd](https://etherscan.io/address/0xBF9226345F601150F64Ea4fEaAE7E40530763cbd) | 20.85 |
| Res | [0x8c5c8d76372954922400e4654AF7694e158AB784](https://etherscan.io/address/0x8c5c8d76372954922400e4654AF7694e158AB784) | 20.85 |
| Harmony | [0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3](https://etherscan.io/address/0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3) | 20.85 |
| Libertas | [0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f) | 20.85 |
| seedlatam.eth | [0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299](https://etherscan.io/address/0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299) | 20.85 |
| 0xRoot | [0xC74392777443a11Dc26Ce8A3D934370514F38A91](https://etherscan.io/address/0xC74392777443a11Dc26Ce8A3D934370514F38A91) | 8.34 |

### Aave Revenue Share Payment

- **Authorization:** [Support Scope Section 9.4.1](https://mips.makerdao.com/mips/details/MIP106#9-4-1-spark-protocol-aave-revenue-share)
- **Proposal:** [Forum post](https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-3-q1-2024/24014)

> If this executive proposal passes, a payment from the Surplus Buffer of **238,339 DAI** to a smart contract ([0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c)) controlled by Aave Governance will be made as part of the Spark Protocol-Aave Revenue Share.

Reviewer: ```suggestion
If this executive proposal passes, a payment from the Surplus Buffer of **238,339 DAI** to a smart contract [0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c) controlled by Aave Governance will be made as part of the Spark Protocol-Aave Revenue Share.
```

---

> title: Template - [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Rewards, Aave-SparkLend Revenue Share, Whitelist new address in the RWA015-A output conduit, USDP Input Conduit Management, Spark Proxy Spell - April 18, 2024
> summary:  Distribute AD compensation for March 2024 and AVC Member participation rewards for Q1 2024, transfer Dai to Aave as part of the SparkLend revenue sharing agreement, Whitelist a new Block Tower address in the RWA015-A output conduit convert USDP to Dai, and deposit to the Surplus Buffer, trigger a Spark proxy spell on Ethereum and Gnosis.
> date: 2024-04-18T00:00:00.000Z
> address: "$spell_address"
> 
> ---
> # [Executive Proposal] March 2024 AD Compensation, Q1 2024 AVC Member Participation Rewards, Aave-SparkLend Revenue Share, Whitelist new address in the RWA015-A output conduit, USDP Input Conduit Management, Spark Proxy Spell - April 18, 2024

Reviewer 2: :yellow_circle: **moderate**

1. Capitalize Titles
2. BlockTower is one word
3. Minor format update

```suggestion
title: Template - [Executive Vote] March 2024 AD Compensation, Q1 2024 AVC Member Participation Rewards, Aave-SparkLend Revenue Share, Whitelist New Address in the RWA015-A Output Conduit, USDP Input Conduit Management, Spark Proxy Spell - April 18, 2024
summary:  Distribute AD compensation for March 2024 and AVC Member participation rewards for Q1 2024, transfer Dai to Aave as part of the SparkLend revenue sharing agreement, whitelist a new BlockTower address in the RWA015-A output conduit, convert USDP to Dai and deposit to the Surplus Buffer, trigger a Spark proxy spell on Ethereum and Gnosis.
date: 2024-04-18T00:00:00.000Z
address: "$spell_address"

---
# [Executive Proposal] March 2024 AD Compensation, Q1 2024 AVC Member Participation Rewards, Aave-SparkLend Revenue Share, Whitelist New Address in the RWA015-A Output Conduit, USDP Input Conduit Management, Spark Proxy Spell - April 18, 2024
```

The Governance Facilitators, SideStream, Dewiz, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.

If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following actions will occur within the Maker Protocol:

- A total of **214.55 MKR** will be distributed to eight Aligned Delegates as compensation for March 2024.
- A total of **175.14 MKR** will be distributed to nine AVC members as participation rewards for Q1 2024.
- A total of **238,339 DAI** will be paid to Aave as part of the SparkLend revenue share.

> - A new Block Tower address will be whitelisted in the RWA015-A output conduit.

Reviewer 2: ```suggestion
- A new BlockTower address will be whitelisted in the RWA015-A output conduit.
```

| Harmony | [0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3](https://etherscan.io/address/0xE20A2e231215e9b7Aa308463F1A7490b2ECE55D3) | 20.85 |
| Libertas | [0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f](https://etherscan.io/address/0xE1eBfFa01883EF2b4A9f59b587fFf1a5B44dbb2f) | 20.85 |
| seedlatam.eth | [0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299](https://etherscan.io/address/0xd43b89621fFd48A8A51704f85fd0C87CbC0EB299) | 20.85 |
| 0xRoot | [0xC74392777443a11Dc26Ce8A3D934370514F38A91](https://etherscan.io/address/0xC74392777443a11Dc26Ce8A3D934370514F38A91) | 8.34 |

### Aave Revenue Share Payment

- **Authorization:** [Support Scope Section 9.4.1](https://mips.makerdao.com/mips/details/MIP106#9-4-1-spark-protocol-aave-revenue-share)
- **Proposal:** [Forum post](https://forum.makerdao.com/t/spark-aave-revenue-share-calculation-payment-3-q1-2024/24014)

If this executive proposal passes, a payment from the Surplus Buffer of **238,339 DAI** to a smart contract [0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c) controlled by Aave Governance will be made as part of the Spark Protocol-Aave Revenue Share.

> ### Whitelist new address in the RWA015-A output conduit

Reviewer 2: ```suggestion
### Whitelist New Address in the RWA015-A Output Conduit
```

- **Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-04-18/24084/3)
- **Proposal:** [Forum post](https://forum.makerdao.com/t/proposed-housekeeping-items-upcoming-executive-spell-2024-04-18/24084)

If this executive proposal passes, the following address will be whitelisted in the `RWA015-A` output conduit:

- [0x6759610547a36E9597Ef452aa0B9cace91291a2f](https://etherscan.io/address/0x6759610547a36E9597Ef452aa0B9cace91291a2f)

> ### Push USDP out of the input conduit

Reviewer 2: ```suggestion
### Push USDP Out of the Input Conduit
```

- USDP balance of MCD_JOIN_PSM_PAX_A increases by **84,211.27 USDP**.
- USDP balance of PSM_PAX_A_INPUT_CONDUIT_JAR decreases to **0 USDP**.
- Surplus Buffer increases by **84,211.27 DAI**.

### Trigger Spark Proxy Spell

- **Authorization:** [Ecosystem Approval](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033/7), Governance Polls [1095](https://vote.makerdao.com/polling/QmfGV2vt), [1096](https://vote.makerdao.com/polling/QmSYZSCQ), [1097](https://vote.makerdao.com/polling/QmcRdMyA), [1098](https://vote.makerdao.com/polling/QmSh8gyC), [1099](https://vote.makerdao.com/polling/QmUhT32b), [1100](https://vote.makerdao.com/polling/QmZND8WW), [1104](https://vote.makerdao.com/polling/QmVXriiT), [1106](https://vote.makerdao.com/polling/QmVsKsGa)
- **Proposal:** [Forum post "[Apr 4, 2024] Proposed Changes to SparkLend for Upcoming Spell"](https://forum.makerdao.com/t/apr-4-2024-proposed-changes-to-sparklend-for-upcoming-spell/24033), [Forum post "SparkLend External Security Access Multisig for Freezer Mom"](https://forum.makerdao.com/t/sparklend-external-security-access-multisig-for-freezer-mom/24070)

> If this executive proposal passes, the SparkLend Proxy Spell on Ethereum and Gnosis at [0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882](https://etherscan.io/address/0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882) will be executed, containing the following items:

Reviewer: ```suggestion
If this executive proposal passes, the SparkLend Proxy Spell on Ethereum at [0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882](https://etherscan.io/address/0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882) will be executed, containing the following items:
```

> If this executive proposal passes, the SparkLend Proxy Spell on Ethereum and Gnosis at [0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882](https://etherscan.io/address/0x3d1DD14Fa08163E7f64b0abf0F514f6276f50882) will be executed, containing the following items:

Reviewer 2: :green_circle: **nit, non-blocking**

1. Technically the Proxy Spell is not *on* Gnosis Chain but is deployed and executed on mainnet with the changes passed through a governance relay.
2. Very minor, but we should agree a consistent styling for the Proxy Spells. Is it a Spark Protocol Proxy Spell? A Spark Proxy Spell? A SparkLend Proxy Spell? et cetera

#### [Ethereum+Gnosis] Upgrade Pool Implementation to Disable Flashloan into Borrow Feature

This item upgrades the SparkLend pool implementation to disable the flashloan-to-borrow feature, reducing unnecessary risks and aligning with the latest Aave V3 release standards for enhanced security.

####  [Gnosis] Onboard sxDAI

This item proposes to onboard sxDAI (Savings xDAI), a yield-enhanced version of sDAI on the Gnosis Chain, as collateral with the following parameters:

* Collateral: Yes
  * Max LTV: 70%
  * Liquidation threshold: 75%
  * Liquidation bonus: 6%
  * Liquidation protocol fee: 10%
* Isolation mode: No
* Efficiency mode: No
* Borrowable: No
* Supply cap: 40,000,000 sxDAI

> * Borrow cap: n/a

Reviewer 2: :green_circle: **nit, non-blocking**

```suggestion
* Borrow cap: N/A
```

#### [Gnosis] Onboard EURe

This item proposes to onboard EURe (Monerium EUR emoney) on the Gnosis Chain with the following parameters:

* Collateral: No

> * Isolation mode: n/a
> * Efficiency mode: n/a

Reviewer 2: :green_circle: **nit, non-blocking**

```suggestion
* Isolation mode: N/A
* Efficiency mode: N/A
```

* Borrowable: Yes
  * Base borrow rate: 0%
  * Optimal borrow rate: 7%
  * Max borrow rate: 57%
  * Optimal Utilization: 90%
  * Reserve factor: 10%
  * Borrowable in isolation mode: No
  * Siloed borrowing: Yes
* Supply cap: 5,000,000 EURe
* Borrow cap: 4,000,000 EURe
* Oracle: EURUSD

#### [Gnosis] Onboard USDC

This item proposes to onboard USDC (Circle USDC) on the Gnosis Chain with the following parameters:

* Collateral: No

> * Isolation mode: n/a
> * Efficiency mode: n/a

Reviewer 2: :green_circle: **nit, non-blocking**

```suggestion
* Isolation mode: N/A
* Efficiency mode: N/A
```

* Borrowable: Yes
  * Base borrow rate: 0%
  * Optimal borrow rate: 12%
  * Max borrow rate: 62%
  * Optimal Utilization: 90%
  * Reserve factor: 10%
  * Borrowable in isolation mode: Yes
  * Siloed borrowing: Yes
* Supply cap: 10,000,000 USDC
* Borrow cap: 8,000,000 USDC
* Oracle: Fixed 1:1 USD pricing

#### [Gnosis] Onboard USDT

This item proposes to onboard USDT (Tether USDT) on the Gnosis Chain with the following parameters:

* Collateral: No

> * Isolation mode: n/a
> * Efficiency mode: n/a

Reviewer 2: :green_circle: **nit, non-blocking**

```suggestion
* Isolation mode: N/A
* Efficiency mode: N/A
```

* Borrowable: Yes
  * Base borrow rate: 0%
  * Optimal borrow rate: 12%
  * Max borrow rate: 62%
  * Optimal Utilization: 90%
  * Reserve factor: 10%
  * Borrowable in isolation mode: Yes
  * Siloed borrowing: Yes
* Supply cap: 10,000,000 USDT
* Borrow cap: 8,000,000 USDT
* Oracle: Fixed 1:1 USD pricing

> #### [Gnosis] Refresh Existing Market Parameter

Reviewer 2: :green_circle: **minor**

```suggestion
#### [Gnosis] Refresh Existing Market Parameters
```
